action.yml

```diff
@@ -1,11 +1,11 @@
 name: 'kube-linter'
-description: 'Scan directory or file with kube-linter'
+description: 'Scan directories or files with kube-linter'
 branding:
   icon: 'check-circle'
   color: 'green'
 inputs:
   directory:
-    description: 'Directory or file to scan'
+    description: 'Directories or files to scan separated by spaces'
```

Comment:
On one hand I like this simple approach of accepting space separated list of dirs but on the other the more natural for yaml will be an array. Can we accept array and us jq to parse it? Does it make sense?